### PR TITLE
Renamed methods, logs and tests and refactored co.rsk.peg.ReleaseTransactionSet.Entry class

### PR DESCRIPTION
**Pull request by nathanieliov**
Renamed methods:

- processReleaseRequests -> processPegoutRequests
- processReleaseTransactions -> processConfirmedPegouts
- addPegoutTxToReleaseTransactionSet -> addPegoutBtcTxToPegoutWaitingForConfirmationsSet
- checkIfEntryExistsInTxsWaitingForSignatures -> checkIfEntryExistsInPegoutsWaitingForSignatures

Also, renamed logs 'release' to 'pegout' on methods renamed and refactored co.rsk.peg.ReleaseTransactionSet.Entry class

### CODE REVIEW

File: rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java [hunk 1]
```diff
@@ -7174,7 +7174,7 @@ private void assertLockingCap(
 
         if (!shouldLock) {
             // Release tx should have been created directly to the signatures stack
-            BtcTransaction releaseTx = provider.getReleaseTransactionSet().getEntries().iterator().next().getTransaction();
+            BtcTransaction releaseTx = provider.getReleaseTransactionSet().getEntries().iterator().next().getPegoutCreationBtcTx();
```
Comment (kelvinator07) [inline rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java:7177]:
```suggestion
            BtcTransaction pegoutBtcTx = provider.getReleaseTransactionSet().getEntries().iterator().next().getPegoutCreationBtcTx();
```

File: rskj-core/src/test/java/co/rsk/peg/BridgeTestIntegration.java [hunk 2]
```diff
@@ -3285,7 +3285,34 @@ void getBtcBlockchainInitialBlockHeight() throws IOException, VMException {
...
+
+    private BtcTransaction createUniqueTransaction(int toPk, Coin value) {
+        return createTransaction(toPk, value, new BtcECKey());
+    }
+
+    private BtcTransaction createTransaction(int toPk, Coin value, BtcECKey btcECKey) {
+        NetworkParameters params = NetworkParameters.fromID(NetworkParameters.ID_REGTEST);
+        BtcTransaction input = new BtcTransaction(params);
+
+        input.addOutput(Coin.FIFTY_COINS, btcECKey.toAddress(params));
```
Comment (kelvinator07) [inline rskj-core/src/test/java/co/rsk/peg/BridgeTestIntegration.java:3307]:
```suggestion
        input.addOutput(Coin.COIN, btcECKey.toAddress(params));
```
I Think We Should Reduce The Values To Reflect Our Daily Transactions Like Values

File: rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java [hunk 3]
```diff
@@ -2593,7 +2593,7 @@ void when_registerBtcTransaction_usesLegacyType_afterFork_notWhitelisted_no_lock
 
         List<BtcTransaction> releaseTxs = provider.getReleaseTransactionSet().getEntries()
```
Comment (kelvinator07) [inline rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java:2594]:
```suggestion
        List<BtcTransaction> pegoutBtcTxs = provider.getReleaseTransactionSet().getEntries()
```

File: rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java (hunk 1 above)
Comment (nathanieliov) [reply rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java]: Done

File: rskj-core/src/test/java/co/rsk/peg/BridgeTestIntegration.java [hunk 4]
```diff
@@ -3285,7 +3285,34 @@ void getBtcBlockchainInitialBlockHeight() throws IOException, VMException {
...
+        return new SimpleBtcTransaction(networkParameters, hash);
+    }
+
+    private BtcTransaction createTransaction(int toPk, Coin value) {
+        return createTransaction(toPk, value, BtcECKey.fromPrivate(BigInteger.valueOf(123456)));
+    }
+
+    private BtcTransaction createUniqueTransaction(int toPk, Coin value) {
+        return createTransaction(toPk, value, new BtcECKey());
+    }
```
[the remark this answers is not in the document]
Comment (nathanieliov) [reply rskj-core/src/test/java/co/rsk/peg/BridgeTestIntegration.java:3301]: Sure. I like the idea.

File: rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java (hunk 3 above)
Comment (nathanieliov) [reply rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java]: Done.

File: rskj-core/src/test/java/co/rsk/peg/BridgeTestIntegration.java (hunk 2 above)
Comment (nathanieliov) [reply rskj-core/src/test/java/co/rsk/peg/BridgeTestIntegration.java]: Done

File: rskj-core/src/test/java/co/rsk/peg/BridgeTestIntegration.java [hunk 5]
```diff
@@ -3292,19 +3293,19 @@ private BtcTransaction createTransaction(Sha256Hash hash) {
         return new SimpleBtcTransaction(networkParameters, hash);
     }
 
-    private BtcTransaction createTransaction(int toPk, Coin value) {
+    /*private BtcTransaction createTransaction(int toPk, Coin value) {
         return createTransaction(toPk, value, BtcECKey.fromPrivate(BigInteger.valueOf(123456)));
-    }
+    }*/
```
[the remark this answers is not in the document]
Comment (nathanieliov) [reply rskj-core/src/test/java/co/rsk/peg/BridgeTestIntegration.java]: Thanks man.

**Review by kelvinator07 — APPROVED** [review]

**Pull request merged**